cmake/BuildLibCURL.cmake

```diff
@@ -206,6 +206,15 @@ macro(curl_build)
     # libcurl-d.a. We use this name below.
     list(APPEND LIBCURL_CMAKE_FLAGS "-DCMAKE_DEBUG_POSTFIX=")
 
+    # Disables 'install' targets.
+    list(APPEND LIBCURL_CMAKE_FLAGS "-DCURL_DISABLE_INSTALL=OFF")
```

Comment:
DISABLE_INSTALL=OFF means ENABLE=ON. Was it expected?

Reply:
Yes. The same value as in Curl's option, we use install target in CMake module [^1]

[^1]: https://github.com/tarantool/tarantool/blob/03c184d4da9be9e1431762989833a6762a704cfb/cmake/BuildLibCURL.cmake#L219